fotoobo/fortinet/fortimanager.py

```diff
@@ -155,7 +193,353 @@ def assign_all_objects(self, adoms: str, policy: str) -> int:
 
         return task_id
 
-    def get_adoms(self, ignored_adoms: Optional[List[str]] = None) -> List[Any]:
+    def delete_adom_address(self, adom: str, address: str, dry: bool = False) -> dict[str, Any]:
+        """
+        Delete an address from an ADOM in FortiManager
+
+        Args:
+            adom:    The ADOM to delete the address from
+            address: The address to delete
+            dry:     Set to True to enable dry-run (no changes on FortiManager)
+
+        Returns:
+            FortiManager result item
+        """
+        result: dict[str, Any] = {}
+        if not dry:
```

Comment:
Please add an empty line before the control block (dito for other methods)

Reply:
Done